tests/audit-tests/src/test/java/uk/gov/di/txma/audit/utilities/ConfigurationReader.java

```diff
@@ -18,5 +14,11 @@ public static String getIPVCoreStubUrl() {
         }
         return IPVCoreStubUrl;
     }
+    public static String getOrchestratorStubUrl() {
+        String orchestratorStubUrl = System.getenv("ORCHESTRATOR_STUB_URL");
```

Comment:
To get the environment variable in AWS, it will have to be called CFN_OrchestrationStubURL